src/flint/test/test_all.py

```diff
@@ -2638,6 +2640,398 @@ def setbad(obj, i, val):
             assert P([1, 2, 1]).integral() == P([0, 1, 1, S(1)/3])
 
 
+def _all_mpolys():
+    return [
+        (flint.fmpz_mpoly, flint.fmpz_mpoly_ctx, flint.fmpz, False),
+        (flint.fmpq_mpoly, flint.fmpq_mpoly_ctx, flint.fmpq, True),
```

Comment:
Is it possible to include multivariate polynomials as a special case along with the univariate polynomial tests?

Perhaps the `test_polys` function would need to be separated into tests that make sense for both vs tests that only make sense in the univariate case.

Reply:
This is certainly possible, however as you noted, there are a number of tests, particularly around interface things like `degree`, `length`, and indexing that just don't agree between the two